config/locales/cy.yml

```diff
@@ -300,6 +300,8 @@ cy:
   categories:
     show:
       view_all: Gweld yr holl ...
+      view_more: View more
```

Comment:
Might be safer just to remove these until we get translations – you'll get warnings in dev mode but it'll just pull the english through on prod.